node/service/src/parachains_db/mod.rs

```diff
@@ -107,6 +113,41 @@ pub fn open_creating(root: PathBuf, cache_sizes: CacheSizes) -> io::Result<Arc<d
 	std::fs::create_dir_all(&path_str)?;
 	upgrade::try_upgrade_db(&path)?;
 	let db = Database::open(&db_config, &path_str)?;
+	let db =
+		polkadot_node_subsystem_util::database::kvdb_impl::DbAdapter::new(db, columns::ORDERED_COL);
+
+	Ok(Arc::new(db))
+}
+
+#[cfg(feature = "full-node")]
+pub fn exists_directory(path: PathBuf) -> bool {
+	match std::fs::metadata(path) {
+		Ok(meta) => meta.is_dir(),
+		Err(_) => false,
+	}
+}
```

Comment:
This is basically `path.is_dir() && path.exists()`

Reply:
:+1: thanks, method after previous change did not make a lot of sense too.